Firestore/core/src/api/optional.h

```diff
@@ -0,0 +1,243 @@
+/*
+ * Copyright 2018 Google LLC
```

Comment:
2021 Google LLC for all new file copyrights

Reply:
Done.